### PR TITLE
Bump 1.4.1

### PR DESCRIPTION
**Pull request by philippjfr**
`panel 1.4.1`

**Destination channel:** defaults

### Links

- [{ticket_number}]() 
- [Upstream repository]()
- [Upstream changelog/diff]()
- Relevant dependency PRs:

### Explanation of changes:

Simple version bump to 1.4.1, also includes markdown as host dependency as it is imported during the build step.

### CODE REVIEW

**Review by skupr-anaconda — DISMISSED** [review]
LGTM but `markdown` in `host`

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,7 +25,8 @@ requirements:
     - setuptools
     - wheel
     # These are also needed at build time.
-    - bokeh >=3.3.0,<3.4
+    - bokeh >=3.4.0,<3.5
+    - markdown =3.6
```
[the remark this answers is not in the document]
Comment (philippjfr) [reply recipe/meta.yaml:29]: Will fix. I vaguely remembered the lint rules requiring a pin.

Comment (philippjfr) [inline recipe/meta.yaml:29]:
```suggestion
    - markdown
```

**Comment by skupr-anaconda** [comment]
An error:
`└─ markdown 3.6.*  does not exist (perhaps a typo or a missing channel).`

The main channel has:
```
conda search markdown

Loading channels: done
# Name                       Version           Build  Channel
markdown                       3.3.4 py310hca03da5_0  pkgs/main
markdown                       3.3.4  py38hca03da5_0  pkgs/main
markdown                       3.3.4  py39hca03da5_0  pkgs/main
markdown                       3.4.1 py310hca03da5_0  pkgs/main
markdown                       3.4.1 py311hca03da5_0  pkgs/main
markdown                       3.4.1 py312hca03da5_0  pkgs/main
markdown                       3.4.1  py38hca03da5_0  pkgs/main
markdown                       3.4.1  py39hca03da5_0  pkgs/main
```

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Comment by cbouss** [comment]
@philippjfr I see bleach is gone from the list of dependencies in pyproject.toml, but is still used in `panel/util/__init__.py`
https://github.com/holoviz/panel/commit/b8c8c345150a6b288860e3739e9aded7bf7a1a57
Can you confirm bleach must be kept in the run section?

**Comment by philippjfr** [comment]
> Can you confirm bleach must be kept in the run section?

Yes, we simply made it lazy so that if a user doesn't ever need it they don't pay the import cost but it is still considered a hard dependency.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -48,7 +49,7 @@ requirements:
     - pandas >=1.2
     - packaging
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:50]: Is this really needed? Should it go in `host` instead?

**Review by JeanChristopheMorinPerso — COMMENTED** [review]
@philippjfr I see that the package on PyPI depends on `xyzservices >=2021.09.1`. Was this intentionally left out of the conda package?

File: recipe/meta.yaml (hunk 2 above)
Comment (philippjfr) [reply recipe/meta.yaml:50]: Yes, it's needed, it's used for version comparisons a bunch.

**Comment by philippjfr** [comment]
>I see that the package on PyPI depends on xyzservices >=2021.09.1. Was this intentionally left out of the conda package?

I believe it's already a dependency of bokeh so shouldn't have to be repeated by us. I think it was a temporary fix because some version of Bokeh hadn't pinned it. I'll remove it in Panel.

File: recipe/meta.yaml (hunk 2 above)
Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:50]: Ok, then you might want to update the dependencies in panel directly :)

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**